pyproject.toml

```diff
@@ -29,6 +29,10 @@ dependencies = ["torch >= 2.0.1", "numpy", "google-cloud-storage", "absl-py"]
 [project.optional-dependencies]
 test = ["pytest"]
 
+lightning = [
```

Comment:
We put the dependencies in "requirements.txt" as well as in the "dependencies" field above. Can we move it there?